doc/CONFIGURE.md

```diff
@@ -0,0 +1,205 @@
+# SSH Slaves plugin
+This plugin allows you to manage agent running on *nix machines over SSH. It adds a new type of agent launch method.
+This launch method will
+
+* Open a SSH connection to the specified host as the specified username.
+* Checks the default version of java for that user.
+* Once it has a suitable version of java, copies the latest remoting.jar via SFTP (falling back to scp if SFTP is not available)
+* Starts the remoting process.
+
+# Prerequisites
+Before you start to use the **SSH Slaves Plugin** with your agents, you need to prepare your agents to run the remoting process,
```

Comment:
You can either make the last comma a semicolon, or end the sentence here and start a new one in the next line.